workflow/tests/ACCA_Examples/Long_Residence.xml

```diff
@@ -52,59 +52,493 @@
           <Spaces>
             <Space>
               <SystemIdentifier id='laundry'/>
-              <FloorArea>77</FloorArea>
+              <FloorArea>77.0</FloorArea>
+              <extension>
+                <DesignLoads dataSource='software'>
+                  <Heating>
+                    <Total/>
+                    <Ducts/>
```

Comment:
@yzhou601 It looks like the HPXML class is writing these values even when they are empty.

Reply:
I haven't figured out why the hvac plant doesn't have the same issue yet, but it seems my next commit fixed it